### ISSUE TITLE
implement `UIWidgetSystem.Checkbox`

### ISSUE THREAD

**Issue by gravestench** [OpenDiablo2/OpenDiablo2#979]
The method should have the signature:
```golang
func (t *UIWidgetSystem) Checkbox(checkedState bool) akara.EID
```

The sprite should be created using `UIWidgetSystem.spriteFactory.Sprite` 

the sprite path and palette should be hardcoded to: 
* d2resource.Checkbox
* d2resource.PaletteFechar

**Comment by ianling** [comment]
I'm looking into this

**Comment by gravestench** [comment]
resolved by #1017
